.github/config.yml

```diff
@@ -0,0 +1,21 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
```

Comment:
This file not related to main purpose of the PR, would you mind removing it?

If you would like to add such probot plugins, feel free to open another PR.